internal/rpc/rpcserver/server.go

```diff
@@ -3469,3 +3471,74 @@ func (s *walletServer) LockAccount(ctx context.Context, req *pb.LockAccountReque
 	}
 	return &pb.LockAccountResponse{}, nil
 }
+// GetPeerInfoResult models the data returned from the getpeerinfo command.
+type GetPeerInfoResult struct {
```

Comment:
don't add this type to the public api. instead move it to a variable below (see later comment)